pkgs/applications/audio/mp3gain/default.nix

```
@@ -1,9 +1,11 @@
 { stdenv, fetchurl, unzip, mpg123 }:
 
 stdenv.mkDerivation {
-  name = "mp3gain-1.6.2";
+  pname = "mp3gain";
+  version = "1.6.2";
+
   src = fetchurl {
-    url = "mirror://sourceforge/mp3gain/mp3gain-1_6_2-src.zip";
+    url = "mirror://sourceforge/mp3gain/${pname}-1_6_2-src.zip";
```

```suggestion
    url = "mirror://sourceforge/mp3gain/${pname}-${builtins.replaceStrings ["."] ["_"] version}-src.zip";
```

I did not test this.

---

This is also out of scope for this PR.